components/dashboard/src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+
+type Props = {
+    percent: number;
+};
+export const ProgressBar: FC<Props> = ({ percent }) => {
+    return (
+        <div className="w-full rounded-full h-3 my-1 bg-gray-600 dark:bg-gray-300">
```

Comment:
**question**: This seems to also resolve the background color issue described in https://github.com/gitpod-io/gitpod/issues/14049 which affects Firefox browser. Could we use the same component there—the logic would have to be reverted for colors though, maybe not worth the effort?

Reply:
Yah we could use it there. Could you explain what you mean by the logic needing to be reverted for the colors to use it on the billing page?

Reply:
@selfcontained The logic comment refers to the colors used on a theme, for example light, are different for the most dashboard elements compared to the toast which is using a revert color, black. This way, colors used for the light theme will not work the same for the toast. Let me know if I can clarify more. Maybe this is won't be a problem with the toast component? 🤷

Reply:
I follow you now, yah I'll give this some thought to figure out a decent approach.